### PR TITLE
add admin session

### PR DESCRIPTION
**Pull request by kian1213**
Hi sir, please check my pull request. Thanks

### CODE REVIEW

**Comment by codegracie** [comment]
Okay

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -1,3 +1,17 @@
 class AdminController < ApplicationController
-  layout "admin"
+  before_action :check_logged_user
+  helper_method :current_user
+
+  def current_user
```
Comment (MuhammadTamzid) [inline app/controllers/admin_controller.rb]: RESTfull 
Read RESTfull pls :)

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<div class="panel panel-primary">
+  <div class="panel panel-heading">
+    Sign in now
```
Comment (MuhammadTamzid) [inline app/views/sessions/new.html.erb]: i18n

File: app/controllers/admin_controller.rb [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+    @current_user ||= User.find_by(id: session[:user_id])
```
Comment (MuhammadTamzid) [inline app/controllers/admin_controller.rb]: Why here??
You already implement this on sessions_helper.rb

File: app/controllers/admin_controller.rb [hunk 4]
```diff
@@ -1,3 +1,9 @@
 class AdminController < ApplicationController
   layout "admin"
+  before_action :correct_user
+
+  private
+  def correct_user
+    redirect_to login_path unless current_user and current_user.admin?
```
Comment (dieunb) [inline app/controllers/admin_controller.rb]: if you use ruby `2.3.0`, you can try `current_user&.admin?`

File: app/controllers/sessions_controller.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
```
Comment (dieunb) [inline app/controllers/sessions_controller.rb]: remove `()`

**Pull request merged**